src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/NotifichePFPage.java

```diff
@@ -24,7 +24,7 @@ public class NotifichePFPage extends BasePage {
     @FindBy(id = "iunMatch")
     WebElement codiceIunTextField;
 
-    @FindBy(id = "filter-notifications-button")
+    @FindBy(id = "filter-button")
```

Comment:
attention at filter for recipient and pa method which already exist

Reply:
Resolve